build-bot-status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import sys
+import time
+import threading
+
+import requests
+
+from light_controller import LightController
+
+
+class BuildBotSession:
+    failColour = "red"
+    busyColour = "orange"
+    goodColour = "green"
+    exceptionColour = "purple"
+
+    def __init__(self, builderURL, slaveName):
+        self.builderURL = builderURL
+        self.slaveName = slaveName
+        self._statusURL = "{}/json/builders/{}".format(self.builderURL,
+                                                       self.slaveName)
+
+        self.lightController = LightController("/dev/ttyACM0",
```

Comment:
1. How do we know that the path is correct, can it be changed later ?
2. Does it make sense to have it somewhere at the beginning of the class for better visibility ?